apps/content-publishing-worker/src/monitor/tx.status.monitor.service.ts

```diff
@@ -46,7 +48,7 @@ export class TxStatusMonitoringService extends BlockchainScannerService {
     private readonly capacityService: CapacityCheckerService,
     @InjectQueue(QueueConstants.PUBLISH_QUEUE_NAME) private readonly publishQueue: Queue,
   ) {
-    super(cacheManager, blockchainService, new Logger(TxStatusMonitoringService.prototype.constructor.name));
+    super(cacheManager, blockchainService, new PinoLogger(getPinoHttpOptions()));
```

Comment:
Why this reversion? Didn't we just clean up all of the `new <some-logger-class>`? We should be injecting a `PinoLogger`, not instantiating a new one.

Reply:
Ah right, I just the NestJS Logger, changed it to Pino, forgetting about your recent update.  Will fix.